2-structured-data/books/model-datastore.js

```diff
@@ -95,9 +95,9 @@ module.exports = function(config) {
       .order('title')
       .start(token);
 
-    ds.runQuery(q, function(err, entities, cursor) {
+    ds.runQuery(q, function(err, entities, nextQuery) {
       if (err) { return cb(err); }
-      var hasMore = entities.length === limit ? cursor : false;
+      var hasMore = entities.length === limit ? nextQuery.startVal : false;
```

Comment:
@stephenplusplus I'm not proud of this. gcloud-node used to use the opaque cursor as the third argument, now that is tucked away in the nextQuery object. I think this is a good step forward, but it makes pagination in a web-app context a little awkward. Is there a better way to do this?